### ISSUE TITLE
object name error in glCopyImageSubData and 	glObjectLabel

### ISSUE THREAD

**Issue by jfactory07** [apitrace/apitrace#469]
Hi there,
The issue is found when I replay a trace file. The srcName and dstName are not redirected using _texture_map or _renderbuffer_map. Since the object name may change during the retrace.

And also glObjectLabel has the similar issue. Please help to fix them.

**Comment by imirkin** [comment]
Probably a duplicate of #404 and #405 ? There's a partial fix for the former which involves always using the texture map (since 99.9999% of uses are with texture objects)

**Comment by jfactory07** [comment]
do you mean the fix in https://github.com/apitrace/apitrace/commit/e99df40414bfa6ec368d48bfaf7824b2a334c651 ?
I don't see any fix using _texture_map  for glCopyImageSubData

**Comment by imirkin** [comment]
Yeah, it's handled by the GLtexture thing as specified in glapi.py. But
it'll always use the texture map, even if you're using a renderbuffer, so
it's an incomplete fix.

**Comment by jrfonseca** [comment]
Yep, this is a dup of #404 and #405.

I don't know when I get around to complete the work I have pending on this, but there's no point in keeping another bug around.   @jfactory07 please subscribe to those bugs instead.
